lib/services/plugins-service.ts

```diff
@@ -240,17 +248,18 @@ export class PluginsService implements IPluginsService {
 			if (cordovaPluginVariables && _.keys(cordovaPluginVariables[pluginData.Identifier]).length === 0) {
 				delete cordovaPluginVariables[pluginData.Identifier];
 			}
-
-			var newCorePlugins = _.without(this.$project.getProperty(PluginsService.CORE_PLUGINS_PROPERTY_NAME, configuration), plugin.toProjectDataRecord());
+			var oldCorePlugins = this.$project.getProperty(PluginsService.CORE_PLUGINS_PROPERTY_NAME, configuration);
+			var newCorePlugins = _.without(oldCorePlugins, plugin.toProjectDataRecord());
 			this.$project.setProperty(PluginsService.CORE_PLUGINS_PROPERTY_NAME, newCorePlugins, configuration);
 			this.$project.saveProject().wait();
```

Comment:
There is no need to save the project or set any of its properties if nothing has changes so these two lines can be moved to the if statement below